ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml

```diff
@@ -0,0 +1,32 @@
+base_images:
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+zz_generated_metadata:
+  branch: ""
+  org: ""
+  repo: ""
+images:
+- dockerfile_path: Dockerfile
+  from: os
+  inputs:
+    openshift_release_golang-1.14:
+      as:
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.14
+  to: route-monitor-operator
+promotion:
+  name: "4.7"
+  namespace: ocp
```

Comment:
We don't need/want a promotion stanza. This is only for cluster operators that are shipped as a part of OCP.